Jenkinsfile

```diff
@@ -1 +1,5 @@
-buildPlugin()
+/*
+ See the documentation for more options:
+ https://github.com/jenkins-infra/pipeline-library/
+*/
+buildPlugin(useContainerAgent: true, platforms: ['linux'], jdkVersions: [11, 17])
```

Comment:
Probably need to keep tests on windows

Reply:
See the ["Improve a plugin" section on Jenkinsfile](https://www.jenkins.io/doc/developer/tutorial-improve/add-a-jenkinsfile/#create-the-jenkinsfile) for the format that is being used more widely to assure Windows and Linux coverage and JDK 11 and 17.